plenum/server/node.py

```diff
@@ -756,19 +771,16 @@ def send_ledger_status_to_newly_connected_node(self, node_name):
 
     def newNodeJoined(self, txn):
         self.setF()
+        new_replicas = self.adjustReplicas()
         if self.adjustReplicas() > 0:
-            self.decidePrimaries()
-        # TODO: Should tell the client after the new node has synced up its
-        # ledgers
-        # self.sendPoolInfoToClients(txn)
+            while new_replicas > 0:
+                self.elector.start_election_for_instance(
+                    self.replicas[-new_replicas].instId)
+                new_replicas -= 1
 
     def nodeLeft(self, txn):
         self.setF()
-        if self.adjustReplicas():
-            self.decidePrimaries()
```

Comment:
Because if you remove a protocol instance there is no need of election, it was wrong before